cli/defenseclaw/commands/cmd_doctor.py

```
@@ -556,6 +641,246 @@ def _check_config(cfg, r: _DoctorResult) -> None:
+                decoded_value = value.decode("utf-8")
+            except UnicodeError:
+                continue
+            normalized = _normalized_gateway_token(decoded_value)
+            if normalized:
+                values[canonical_key] = normalized
+    except OSError:
+        return {}
+    return values
+
+
+def _gateway_data_dir_integrity_problem(cfg) -> str:
```

Ruff (0.16.0)

[warning] Too many return statements (10 > 6)

(PLR0911)